Dockerfile

```diff
@@ -14,48 +16,50 @@
 
 # Build the binary
 FROM golang:1.19 AS builder
-
 WORKDIR /workspace
+
+# Install dependencies.
+RUN apt-get update && apt-get install -y jq && mkdir bin
+
+# Install kubectl.
+ARG kubernetes_version=1.24.2
```

Comment:
(as a follow-up) is it possible to run `go mod` to get this value dynamically?

Reply:
Yeah we can do that with grep and whatnot, haven't found a clean way to set it

Reply:
```
$ go list -m -json k8s.io/kubernetes | jq .Version
"v1.24.3"
```